README.md

```diff
@@ -275,6 +280,176 @@ Reads if device geolocation is activated or not (read-only).
+
+List, filter and paginate patches.
+⚠️ Restricted to **private** API keys, **do not expose it to the browser**.
+
+```js
+// get all patches, paginated
+pk.patch.list().then((res) => {
+  console.log(res.results);
+});
+
+// filter and paginate patches
+pk.patch.list({
+  query: 'angeles',
+  countries: ['us'],
+  types: ['street'],
+  status: 'approved',
+  maxResults: 10,
+  offset: 0,
+}).then((res) => {
+  console.log(res.results);
+});
+```
+
+| Parameter | Type | Description |
+| --- | --- | --- |
+| `query` | `string?` | Search query terms. |
+| `opts` | `key-value mapping` (optional) | Search options. |
+| `opts.status` | `('all' | 'pending' | 'approved')?` | Publication status. |
+| `opts.countries` | `string[]?` | Countries filter, array of [two-letter ISO](https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2) country codes. |
+| `opts.types` | `string[]?` | Types filter, array of accepted values: `street`, `city`, `airport`, `bus`, `train`, `townhall`, `tourism`. Prepend `-` to omit a type like `['-bus']`. Unset to return all. |
+| `opts.maxResults` | `number?` | Maximum number of results to return. |
+| `opts.offset` | `number?` | Paginate results starting from the offset. |
+
+### `pk.patch.create()`
+
+Add a missing record or fix an existing one.
+⚠️ Restricted to **private** API keys, **do not expose it to the browser**.
+
+```js
+// Adding a missing record
+const record = {
+  type: 'street',
+  name: 'Example street',
+  city: 'Los Angeles',
+  county: 'Los Angeles',
+  administrative: 'California',
+  country: 'United States of America',
+  countrycode: 'us',
+  coordinates: '33.9955095,-118.472482',
+  zipcode: [90291],
+  population: 3849000, // optional
+};
+pk.patch.create(record, { status: 'approved' }).then((record) => {
+  console.log(record);
+});
+
+// Fixing an existing record
+pk.patch.create(
+  { population: 3849000 },
+  originalRecord, // original record from `pk.search` or `pk.reverse`
+  { status: 'approved' }
+).then((record) => {
+  console.log(record);
+});
+```
+
+| Parameter | Type | Description |
+| --- | --- | --- |
+| `update` | `key-value mapping` | Full record if adding, only updated fields if fixing. |
+| `update.type` | `string` | One of `airport`, `bus`, `city`, `street`, `tourism`, `townhall`, `train`. |
+| `update.name` | `string` | Record display name (street name, city name, station name...). |
+| `update.city` | `string` | Record city name. |
+| `update.county` | `string` | Record county/province/department. |
+| `update.administrative` | `string` | Record administrative/region/state. |
+| `update.country` | `string` | Record country name. |
+| `update.countrycode` | `string` | Record [two-letter ISO](https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2) country code. |
+| `update.coordinates` | `string` | Record coordinates in format `lat,lng`. |
+| `update.zipcode` | `string[]` | Record postal/zip code(s). |
+| `update.population` | `number?` | Record population of its city. |
```

Comment:
`population` is required from the API

Reply:
I thought we agreed on making it optional. I'll update it, my bad.

Reply:
Nope, we said OK to be optional in the admin (you probably default to 0 or something), but the API requires it anyway.